experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -330,6 +330,9 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
       'response',
       (response: http.IncomingMessage & { aborted?: boolean }) => {
         this._diag.debug('outgoingRequest on response()');
+        if (request.listenerCount('request') <= 0) {
```

Comment:
```suggestion
        if (request.listenerCount('response') <= 0) {
```

Naive Q: should that eventName be `response`?